.travis.yml

```diff
@@ -1,3 +1,8 @@
 language: ruby
+script: bundle exec rake test close_old_pull_requests
```

Comment:
this makes it look like we're adding `rake test` as well as `rake close_old_pull_requests`. I'm assuming that there was just an implied `script: bundle exec rake test` previously because we didn't supply any line at all? If so it might be useful to make that version explicit first as a separate commit, so that it's more obvious that _this_ commit is only adding the `close_old_pull_requests` part.